at_secondary/at_secondary_server/lib/src/notification/resource_manager.dart

```diff
@@ -11,57 +10,87 @@ import 'package:at_utils/at_logger.dart';
 /// Class that is responsible for sending the notifications.
 class ResourceManager {
   static final ResourceManager _singleton = ResourceManager._internal();
-  bool _isRunning = false;
+  bool _isProcessingQueue = false;
 
-  bool get isRunning => _isRunning;
+  bool get isProcessingQueue => _isProcessingQueue;
+
+  bool _isStarted = false;
+  bool get isStarted => _isStarted;
+
+  bool _nudged = false;
 
   ResourceManager._internal();
 
   factory ResourceManager.getInstance() {
     return _singleton;
   }
 
+  void init(int? outboundConnectionLimit) {
+    NotifyConnectionsPool.getInstance().init(outboundConnectionLimit);
+    _isStarted = true;
+    Future.delayed(Duration(milliseconds: 0)).then((value) {
```

Comment:
if milliseconds is 0..is there a difference in calling _schedule directly vs calling inside Future.delayed(...) ?

Reply:
Anecdotally, I believe I've observed a difference but I don't have hard proof. I think it's different in that the code as I've written it creates a new event for the Isolate's event queue